### DISCUSSION TITLE
How to use the E28-2G4M20S 100mw as a TX

### DISCUSSION THREAD

**Discussion by vinvaa** [ExpressLRS/ExpressLRS#1521] [Q&A]
I have an E28-2G4M20S RF module with 100mw of power, and want to replace the default 27S in DIY-2400-TX-ESP32-E28 (support 250mw), so what is the value value in the target file, and what is the corresponding value of each registration?

**Comment by Lance0724** [comment]
U can  use it replace E28-2G4M27S.  
Just need change the power_max and power_hign  in souce code.
(src/hardware/TX/DIY 2400 E28.json)
